Server/Controls/RegistryTreeView.cs

```diff
@@ -8,11 +8,20 @@ namespace xServer.Controls
 {
     public class RegistryTreeView : TreeView
     {
-
         public RegistryTreeView()
         {
             //Enable double buffering and ignore WM_ERASEBKGND to reduce flicker
             SetStyle(ControlStyles.OptimizedDoubleBuffer | ControlStyles.AllPaintingInWmPaint, true);
         }
+
+        protected override CreateParams CreateParams
+        {
+            get
+            {
+                CreateParams handleParam = base.CreateParams;
+                handleParam.ExStyle |= 0x02000000;// WS_EX_COMPOSITED       
```

Comment:
Hm, how does this fix the flickering?
And btw. can you remove the spaces behind the comment? xD